tests/scripts/convert_utest.py

```diff
@@ -1488,7 +1488,6 @@ def setUp(self) -> None:
         self.replacement_values = [
             ("${VE_suit}", "Validation & Encoding"),
             ("${VE_VE2_desc}", "You have invented a new attack against Data Validation and Encoding"),
-            ("${VE_VE2_misc'}", "Read more about this topic in OWASP's free Cheat Sheets"),
```

Comment:
Is the _misc never called?

Reply:
This is not really testing misc it’s testing how templating is suppose to work if the template label has the wrong case, contain apostroph or the template value is surrounded with whitespace. I removed it as this functionality is not used and make the code complicated and error prone.